readme2tex/render.py

```diff
@@ -25,6 +25,16 @@
 except NameError:
     pass
 
+html_escape_dict = {
+    '"': '&quot;',
+    '\n': '&#x0a;',
+    '<': '&lt;',
+}
+
+def replace_all(text, dic):
+  for i, j in dic.iteritems():
+    text = text.replace(i, j)
+  return text

```

Comment:
```python
from xml.sax.saxutils import quoteattr
```
would probably be more idiomatic here. Note that this includes the open and closing quotation marks as well.